CMakeLists.txt

```diff
@@ -303,9 +314,10 @@ SET(CPACK_SOURCE_STGZ "OFF")
 SET(CPACK_SOURCE_TBZ2 "ON")
 SET(CPACK_SOURCE_TGZ "OFF")
 SET(CPACK_SOURCE_TZ "OFF")
+SET(CPACK_PACKAGE_CONTACT "team@opencog.org")
```

Comment:
I don't think this mailing list exists. I think you want to use <opencog@googlegroups.com> as the contact point.